src/Components/Inputs/Composed/TimeInput.tsx

```diff
@@ -17,7 +17,7 @@ import {
 
 const TIME_INPUT_DROPDOWN_SIZE = {
   xs: '60px',
-  sm: '66px',
+  sm: '100px',
   md: '70px',
   lg: '90px',
```

Comment:
The only thing I am concerned about is that `sm` is now the biggest size.

Reply:
I was about to comment about it, this component in't used anywhere so we never caught that the different component sizes were never used. 

But now that I am in the component, let me just fix that.